index.html

```diff
@@ -337,17 +333,18 @@ <h2 class="contact-title">Contact me</h2>
                 that
                 needs coding.
                 I’d love to help with it!</p>
-            <form action="">
+            <form method="POST" action="https://formspree.io/f/mnqywdbe">
                 <div class="input-container">
-                    <input type="text" placeholder="Name">
+                    <input id="full-name" name="fullName" required maxlength="30" type="text" placeholder="Name">
                 </div>
 
                 <div class="input-container">
-                    <input type="text" placeholder="Email">
+                    <input id="email" name="email" required type="text" placeholder="Email">
```

Comment:
- [ ] Kindly make sure your email `input`  has the `type="email"` instead of `type="text"` in order to match the project requirements.